tests/ipc_sock.test

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+#
+# Run the IPC tests under the stat wrapper,
+# this simulates /etc/libqb/use-filesystem-sockets existing
+# so we can test both options without breaking other things
+# that might be running on this system
+#
+if [ "$(uname -s)" = "Linux" ]
+then
+  if [ -f $(pwd)/.libs/libstat_wrapper.so ]
+  then
+    export LD_PRELOAD=$(pwd)/.libs/libstat_wrapper.so
```

Comment:
best to have:
export LD_PRELOAD="$(pwd)/.libs/libstat_wrapper.so" in case pwd contains spaces... it doesn´t happen in CI or any sane systems.. but it can happen :)